query.py

```diff
@@ -0,0 +1,394 @@
+import json
+import requests
+import networkx as nx
+
+
+class Query:
+    def __init__(
+        self,
+        service_base_url: str,
+        CapabilityStatement_path: str = None,
+    ) -> None:
+        self.base = service_base_url
+        self.possible_references = self._get_rev_include_possibilitie(
+            CapabilityStatement_path
+        )
+        """Instantiate the class and create the query object
+
+        Arguments:
+            service_base_url {str} -- the Service Base URL
+            CapabilityStatement_path {str} -- path to the json file that
```

Comment:
I know it's fhir specific but can you also tell very quickly what a CapabilityStatement is and looks like?